Dockerfile

```diff
@@ -3,37 +3,33 @@ FROM ubuntu:20.04
 RUN groupadd -g 5000 sync-engine \
   && useradd -d /home/sync-engine -m -u 5000 -g 5000 sync-engine
 
-ENV TZ="Etc/GMT"
-ENV DEBIAN_FRONTEND=noninteractive
+ENV \
+  TZ="Etc/GMT" \
+  LANG="en_US.UTF-8" \
+  LC_ALL="en_US.UTF-8" \
+  DEBIAN_FRONTEND=noninteractive \
+  PATH="/opt/venv/bin:$PATH"
+
 ARG BUILD_WEEK=0
 RUN echo $BUILD_WEEK && apt-get update \
   && apt-get dist-upgrade -y \
-  && apt-get install -y \
+  && apt-get install --no-install-recommends -y \
     tzdata \
-    build-essential \
+    locales \
+    make \
     curl \
-    dnsutils \
+    gpg \
+    gpg-agent \
+    dirmngr \
     gcc \
-    g++ \
     git \
     python3-dev \
     python3-pip \
-    wget \
     gettext-base \
-    language-pack-en \
-    libcurl4-openssl-dev \
     libmysqlclient-dev \
-    libxml2-dev \
-    libxslt-dev \
-    libxslt1-dev \
     mysql-client \
-    pkg-config \
-    lsof \
-    net-tools \
-    shared-mime-info \
-    telnet \
     vim \
-    libffi-dev \
+  && locale-gen en_US.UTF-8 \
```

Comment:
We used to install `language-pack-en`, that just includes all the English variants around the world, but we only need `en_US.UTF-8`